template/package.json

```diff
@@ -93,6 +93,9 @@
     "vue-style-loader": "^3.0.1",
     "vue-template-compiler": "^2.3.3",
     "webpack": "^2.6.1",
+    {{#cdn}}
+    "webpack-cdn-plugin": "^1.3.4",
```

Comment:
Won't this end up being an empty line when `cdn` is false and two empty lines when `cdn` is true?